### DISCUSSION TITLE
How to bind byte[] property of MyOption from appsettings.json?

### DISCUSSION THREAD

**Discussion by LeaFrock** [dotnet/runtime#46572] [Q&A] [answered]
Details see #36034

I find that **main** branch includes the codes but **release\5.0** branch doesn't.

So, when will it be available? .NET 5.1 or .NET 6?
What can I do for it now?

**Comment by PathogenDavid** [comment, marked as answer]
> So, when will it be available? .NET 5.1 or .NET 6?

I don't believe there's been any indication that .NET 5.1 will be a thing. So you're probably looking at .NET 6 which I would not expect before November 2021.

> What can I do for it now?

As noted in the linked issue you can use a string property and do the base64 conversion yourself.
